### PR TITLE
enable JDK 9

### PR DESCRIPTION
**Pull request by vladak**
try to enable JDK 9

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,8 +33,27 @@ jobs:
...
+    - stage: test
+      name: macOS + Oracle JDK 8
+      os: osx
+      osx_image: xcode9.3
+      install: true
+      script: dev/main
+      before_install: dev/before_install
+      before_script: dev/before
+    - stage: test
+      name: macOS + Oracle JDK 10
```
Comment (tulinkry) [inline .travis.yml]: why not linux and jdk 10?

Comment (vladak) [reply .travis.yml]: see https://docs.travis-ci.com/user/reference/trusty/#jvm-clojure-groovy-java-scala-images

Comment (vladak) [reply .travis.yml]: for macOS, I wanted to see if JDK 10 works at all, it does not have to be there.

Comment (tulinkry) [reply .travis.yml]: aha, ok not supported.

**Pull request merged**